dataspace-connector/full/docker-compose.yml

```diff
@@ -56,9 +56,31 @@ services:
       - configmanager
     restart: always
 
+  keycloak:
+    depends_on:
+      - postgres

```

Comment:
Just out of curiosity:
1. is keycloak really dependent on postgres?
2. if yes, is the depends_on command sufficient? Background: depends_on only checks if the container itself is initially started, but not if it is completely started and certain API can be called (that's why there are checks for DSC connectivity at startup in CM v7).
3. if no, another solution must be implemented

Reply:
Keycloak needs an underlying database, such as postgres or mysql.
I think depends_on is sufficient, keycloak also uses depends_on on their example docker-compose files.

https://github.com/keycloak/keycloak-containers/blob/master/docker-compose-examples/keycloak-postgres.yml